pdd/trainer.py

```diff
@@ -0,0 +1,169 @@
+from .metrics import knn_acc
+from .plot import plot
+import numpy as np
+import torch
+from tqdm import tqdm
```

Comment:
and again: 
1) pdd imports should after other 
From PEP8:

Imports should be grouped in the following order:

    Standard library imports.
    Related third party imports.
    Local application/library specific imports.

You should put a blank line between each group of imports.

2) Use either relative (.metrics) or absolute (pdd.metrics) imports - absolute preferred